ledger/sandbox-common/src/main/scala/platform/sandbox/cli/CommonCliBase.scala

```diff
@@ -343,11 +343,12 @@ class CommonCliBase(name: LedgerName) {
           "Maximum command deduplication duration."
         )
 
-      opt[Unit]("use-self-service-error-codes")
+      opt[Unit]("use-legacy-grpc-error-codes")
```

Comment:
```suggestion
      opt[Unit]("disable-self-service-error-codes")
```
or
```suggestion
      opt[Unit]("use-pre-1.18-error-codes")
```

Reply:
Hmm, I am not sold on any of those:
* `disable-self-service-error-codes` mentions self service which we deemed to keep an internal term. For the users it's just error codes
* `use-pre-1.18-error-codes` has a dot in it 🙃 and a version. It seems unusual

How about `grpc-status-codes-compat-mode`?

Reply:
`use-pre-1.18-error-codes` happens to align well with our documentation. I like it.

Reply:
Makes sense, however, with 'compatibility mode' the question is 'compatible with what?' The SDK version would make that fairly explicit.
Assuming we'll deprecate this flag for Daml SDK 2.0, I don't have a strong opinion on what we choose, any of the above should work.

Reply:
Fine then. 2ff8c8c54136c655e75bc84245fa3ea30998f5e9